rhoas-cli-service-registry/README.adoc

```diff
@@ -243,40 +265,34 @@ Artifacts might include, for example, schemas that define the structure of Kafka
 [NOTE]
 ====
 You can use `rhoas service-registry use` to switch to a specific {registry} instance.
-
-.Selecting a {registry} instance to use
-[source,shell,subs="attributes"]
```

Comment:
Line 248-252: Did you mean to remove this code entirely, or to split the command from the output?

Reply:
Like before, I removed it since it's redundant.